hoptimator-operator/src/main/java/com/linkedin/hoptimator/operator/HoptimatorOperatorApp.java

```diff
@@ -59,9 +61,15 @@ public static void main(String[] args) throws Exception {
 
     Options options = new Options();
 
-    Option namespace = new Option("n", "namespace", true, "specified namespace");
-    namespace.setRequired(false);
-    options.addOption(namespace);
+    Option defaultNamespace = new Option("n", "namespace", true,
+        "default namespace to configure K8s API");
+    defaultNamespace.setRequired(false);
+    options.addOption(defaultNamespace);
+
+    Option watchNamespace = new Option("w", "watch", true,
```

Comment:
This does make sense. Let's keep the watch argument, and just pass this through to the `informers`.